run_dacapo.py

```diff
@@ -0,0 +1,141 @@
+        "batik",
+        # java.lang.IllegalStateException: Unable to acquire application service.
+        # Ensure that the org.eclipse.core.runtime bundle is resolved and started.
+        "eclipse",
+        # URL /examples/jsp/jsp2/el/basic-arithmetic.jsp returned status 500 (expected 200)
+        "tomcat",
+    ],
+    "graal-ce": [
+        # java.lang.NoClassDefFoundError: com/sun/image/codec/jpeg/TruncatedFileException
+        "batik",
+        # java.lang.IllegalStateException: Unable to acquire application service.
+        # Ensure that the org.eclipse.core.runtime bundle is resolved and started.
+        "eclipse",
+        # URL /examples/jsp/jsp2/el/basic-arithmetic.jsp returned status 500 (expected 200)
+        "tomcat",
+        # javax.ejb.FinderException: Cannot find account forPRGS
+        "tradesoap",
+    ],
+    "openj9": [
+        # java.lang.NoClassDefFoundError: com.sun.image.codec.jpeg.TruncatedFileException
+        "batik",
+        # java.lang.IllegalStateException: Unable to acquire application service.
+        # Ensure that the org.eclipse.core.runtime bundle is resolved and started.
+        "eclipse",
+        # URL /examples/jsp/jsp2/el/basic-arithmetic.jsp returned status 500 (expected 200)
+        "tomcat",
+        # java.lang.ArrayIndexOutOfBoundsException: Array index out of range: -12
+        "tradebeans",
+        # java.lang.ArrayIndexOutOfBoundsException: Array index out of range: -14
+        "tradesoap",
+    ]
+}
+
+# Each entry is a list of arguments used to invoke the VM.
+JAVA_VMS = OrderedDict()
+JAVA_VMS["graal-ce-hotspot"] = [os.path.join(PATHS["GRAALCE_DIR"],
+                                             "bin", "java")]
+JAVA_VMS["graal-ce"] = [os.path.join(PATHS["GRAALCE_DIR"], "bin", "java"),
+                        "-XX:-EnableJVMCI", "-XX:-UseJVMCICompiler"]
+JAVA_VMS["openj9"] = [os.path.join(PATHS["OPENJ9_DIR"], "bin", "java")]
+
+
+def main():
+    for jvm_name, jvm_args in JAVA_VMS.items():
+        csvp = "dacapo.%s.results" % jvm_name
+        with open(csvp, 'w') as csvf:
+            writer = csv.writer(csvf)
+            writer.writerow(
+                ['processnum', 'benchmark'] + list(range(ITERATIONS)))
+
+            for benchmark in BENCHMARKS:
+                if benchmark in BROKEN[jvm_name]:
+                    print("!!! %s::%s -- Skipping all. Marked broken." %
+                          (jvm_name, benchmark))
+                    sys.stdout.flush()
+                    continue
+
+                for pexec_idx in range(PROCESSES):
+                    print(">>> %s::%s::%s" %
+                          (jvm_name, benchmark, str(pexec_idx)))
+                    sys.stdout.flush()
+
+                    # Flush the CSV writing, and then give the OS some time to
+                    # write stuff out to disk before running the next process
+                    # execution.
```

Comment:
I was expecting to see a `sleep` (or similar) given the "give the OS some time" comment. Am I missing out on something obvious?

Reply:
I removed it, since we aren't using these standalone runners to collect real data. Only to check if the benchmarks work.

For real results, I will be updating the Krun config file.